starboard/linux/shared/test_filters.py

```diff
@@ -24,6 +24,7 @@
     'nplb': [
         'MultiplePlayerTests/*/*sintel_329_ec3_dmp*',
         'MultiplePlayerTests/*/*sintel_381_ac3_dmp*',
+        'PosixDirectoryCanOpenTest.FailureMissingStaticContent',
```

Comment:
Should we create a follow up bug for this test? Will it be fixed once we do another merge from trunk or is it something actually going wrong with the feature branch's build?

Reply:
I think this was already removed from trunk, so it should go away when we merge back.